### ISSUE TITLE
Connection lost immediately on Mac

### ISSUE THREAD

**Issue by AndrewMHenry** [zauberzeug/nicegui#1154]
### Description

# 1. What are you trying to do?

On my Mac with the latest version of NiceGUI, I'm having trouble running the following "Hello, World"-type program:
```
from nicegui import ui

ui.label("Hello")

ui.run()
```
I am running it in a test virtual environment with just `nicegui` installed into it:
- NiceGUI version: 1.2.24
- Python version: CPython 3.10.6
- OS: macOS Monterey, Version 12.6
- Processor: 2.3 GHz 8-Core Intel Core i9

# 2. What do you expect to happen?

The webpage should come up without any error messages.

# 3. What happens instead?

The webpage comes up, but immediately shows the box with *Connection lost.*

# Other notes

The program works when I replace `ui.run()` with `ui.run(native=True)`, and I can also run a NiceGUI program on a remote Ubuntu Linux VM and access the webpage successfully from my local machine using port forwarding.

Is anyone else having the same problem? I still have the problem with old versions of NiceGUI, so it doesn't seem like it's due to a recent NiceGUI change.

**Comment by rodja** [comment]
Hello @AndrewMHenry. That should not happen. We develop on Mac without problems. Please make sure you have the right version:

```py
from nicegui import ui, __version__

ui.label(f"{__version__=}")

ui.run()
```

And maybe try reinstalling the package. You could also try another port with `ui.run(port=5500)` or similar and check your firewall. If the error persists you may want to post the output from Python and the browser debug console error log.

**Comment by AndrewMHenry** [comment]
Hi @rodja , thanks for the advice! I confirmed that `__version__='1.2.24'`, but changing the port number to `5500` fixed the problem completely and (`8081` also worked).

So far, I've only had the problem on ports `8080` and `80`. As a test, I tried running
```
python -m http.server 8080
```
and I could access the server from my browser without problems (same with port `80`). So, it seems like Python can serve on those ports, and my browser can access them.

It's also interesting that serving up a NiceGUI app on `8080` does connect temporarily, rather than failing to connect in the first place. That makes me think it's not a permission issue, but does anyone have thoughts on what else it could be?

**Comment by rodja** [comment]
> It turns out that there were other IT-type processes running on those ports; is it plausible that those processes are causing NiceGUI to disconnect without preventing it from connecting in the first place?

No, not to my knowledge. If the port is already used, the NiceGUI server startup should fail and report the blocked port. What was the other "IT-type process"? How to reproduce?
